src/handlers/motions/motionCreated/handlers/editColony.ts

```diff
@@ -0,0 +1,19 @@
+import { TransactionDescription } from 'ethers/lib/utils';
+import { ContractEvent, motionNameMapping } from '~types';
+import { getPendingMetadataDatabaseId } from '~utils';
+import { createMotionInDB } from '../helpers';
+
+export const handleEditColonyMotion = async (
+  event: ContractEvent,
+  { name }: TransactionDescription,
+): Promise<void> => {
+  const { transactionHash, contractAddress: colonyAddress } = event;
+  const pendingColonyMetadataId = getPendingMetadataDatabaseId(
+    colonyAddress,
+    transactionHash,
+  );
+  await createMotionInDB(event, {
+    type: motionNameMapping[name],
+    pendingColonyMetadataId,
```

Comment:
Here we're following the same pattern as with Domains. We attach the "pending" metadata to the action, so it can be retreived when the motion is finalized.